### PR TITLE
Update my_api.yaml

### PR DESCRIPTION
**Pull request by rajshruti18**

### CODE REVIEW

File: api/openapi/my_api.yaml [hunk 1]
```diff
@@ -446,34 +454,7 @@ components:
...
-                  legalValues:
-                    description: "The 'legalValues' will always be present in the '/config' endpoint response and need not be provided to or from the client in the '/guidance' endpoint call. Use the JSON schema, https://json-schema.org/draft/2019-09/json-schema-validation.html. For example, { type: 'string', 'enum': [ 'Gentamicin', 'Amikacin' ] } to define an enumeration,  { 'type': 'number' } for a float, and { 'type': 'number', 'minimum': '0' } for a positive number, and { 'type': 'integer' } for an integer. NOTE: This value is actually an object type, but we use 'string' here to overcome a shortcoming of the swagger ui. This design choice was made in order to prioritize a better interactive documentation experience for the API user. "
-                    type: string
-                    example: {"type": "number", "minimum": "0"}
-                  timestamp:
-                    type: string
-                    format: date-time
-                    description: "If not specified, assume current time"
+                  $ref: '#/components/schemas/PatientVariableValue/properties'
```
Comment (hyi) [inline api/openapi/my_api.yaml]: @rajshruti18 I am not sure whether swagger API allows to reference properties directly, so I pasted this updated my_api.yaml file into https://editor.swagger.io site and it does not pass swagger validation, and give a structural error as follows: ```Structural error at components.schemas.Guidance.properties.justification.properties.inputs.items.properties.$ref should be object```. So it looks like we can only reference objects as a whole. It looks like the only way we can consolidate the PatientVariableValue object to be used by both ```UserSuppliedPatientVariables``` and ```inputs``` in ```justification``` in guidance request response is to relax the why field from required to optional in ```inputs``` object definition in `justification` in guidance request response. We will need input from @krobasky on whether this is a good idea or not.

Comment (rajshruti18) [reply api/openapi/my_api.yaml]: I thought this would happen. Waiting for reply from @krobasky .

Comment (krobasky) [reply api/openapi/my_api.yaml]: Yes, I think its ok to relax the why field to optional so that the same schema can be used across other PDS APIs.

Comment (rajshruti18) [reply api/openapi/my_api.yaml]: @hyi Relaxed the requirement of why field. Please review it and let me know if I can merge it.

**Review by hyi — APPROVED** [review]

**Pull request merged**